doc/manual/faq.rst

```diff
@@ -1,110 +1,171 @@
 
-Use the dot (".") in dataset names to separate folders. The GUI will automatically create and delete folders in the dataset tree display.
+Minor version mismatches are common even in purchased crates: M-Labs flashes new crates with the most up-to-date binaries, but ARTIQ is always being worked on and may be updated while the crate is in shipment. As long as you are using the stable version of ARTIQ, this is rarely the direct cause of any issues. Breaking changes are normally strictly limited to new release versions, or to the beta branch, which makes no promises of stability. Updates that *are* applied to the stable version are usually bug fixes, documentation improvements, or other quality-of-life changes. Nonetheless, it's a good idea to keep your gateware and software up to date with each other.
 
-organize experiment windows in the dashboard?
----------------------------------------------
+change configuration settings of satellite devices?
+---------------------------------------------------
 
-Experiment windows can be organized by using the following hotkeys:
+Currently, it is not possible to reach satellites through ``artiq_coremgmt config``, although this is being worked on. On Kasli, use :class:`~artiq.frontend.artiq_mkfs` and :class:`~artiq.frontend.artiq_flash`; on Kasli-SoC, preload the SD card with a ``config.txt``, formatted as a list of ``key=value`` pairs, one per line.
 
-* CTRL+SHIFT+T to tile experiment windows
-* CTRL+SHIFT+C to cascade experiment windows
+Don't worry about individually flashing idle or startup kernels. If your idle or startup kernel contains subkernels, it will automatically compile as a ``.tar``, which you only need to flash to the master.
 
-The windows will be organized in the order they were last interacted with.
+.. _faq-networking:
 
-write a generator feeding a kernel feeding an analyze function?
----------------------------------------------------------------
+troubleshoot networking problems?
+---------------------------------
 
-  Like this::
+Diagnosis aids:
 
-    def run(self):
-        self.parse(self.pipe(iter(range(10))))
+    - Can you ``ping`` the device?
+    - Is the Ethernet LED on?
+    - Is the ERROR LED on?
+    - Is there anything unusual recorded in :ref:`the UART log <connecting-UART>`?
+
+Some things to consider:
+
+    - Is the ``core_addr`` field of your ``device_db.py`` set correctly?
+    - Are your core device's IP address and networking configurations definitely set correctly? Check the UART log for evidence of this, and ask your network administrator about the other settings.
+    - Is your core device configured for an external reference clock? If so, it cannot function correctly without one. Is the external reference clock plugged in?
+    - Are SFP0 and Ethernet plugged in all the way? Are they working? Try different cables and SFP adapters; M-Labs tests with CAT6 cables, but lower categories should be supported too.
```

Comment:
SFP adapter is only applicable to Kaslis, Kasli-SoCs use the Ethernet port, I'd make that distinction.